internal/broker/plans_schema.go

```diff
@@ -9,6 +9,14 @@ import (
 	"github.com/kyma-project/kyma-environment-broker/internal/networking"
 )
 
+const (
+	haAutoscalerMinimumValue       = 3
```

Comment:
Maybe we could reuse `minAutoScalerMinValueForHA` from runtimes package as this value is already defined there and the package is imported here.

Reply:
Done, I replaced the const in broker package with the one in runtime package, namely `HAAutoscalerMinimumValue`